src/test/kotlin/no/nav/modiapersonoversikt/skrivestotte/service/LeaderElectorServiceSpec.kt

```diff
@@ -24,4 +24,10 @@ internal class LeaderElectorServiceSpec {
         val service = LeaderElectorService(Configuration(electorPath = electorPath, clusterName = "something else"))
         assertEquals("inethostname", service.getLeader().name)
     }
+
+    @Test
+    fun `Sjekke om ignorerer ukjent variabel`() {
+        val service = LeaderElectorService(Configuration(electorPath = electorPath))
+        assertEquals(LeaderElectorResponse(name="inethostname"), service.getLeader())
```

Comment:
Tenker vi kan justere litt på denne. For det er ikke klart og tydelig hvorfor denne tester at ting fungerer med ett ukjent felt i json-responsen.

Kanskje noe alà;
```suggestion
        assertDoesNotThrow {
            "{\"name\":\"inethostname\", \"unknownfield\":\"othervalue\"}".fromJson<LeaderElectorResponse>()
        }
```